### PR TITLE
Feat: positive token rebase limiter fuzz tests

### PR DESCRIPTION
**Pull request by TheDZhon**
Fuzz tests for the `PositiveTokenRebaseLimiter` library:
- [x] New harness contract
- [x] Remove outdated mock from `0.8.9/test_helpers`
- [x] Fuzzed tests for all functions 
- [x] File an issue about precision loss: https://github.com/lidofinance/lido-dao/issues/835 

---

:warning:  Note that during the fuzzing design and trials a precision loss weakness has been [detected](https://github.com/lidofinance/lido-dao/issues/835).

### CODE REVIEW

File: test/0.8.9/positiveTokenRebaseLimiter.t.sol [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+    _fuzzData.preTotalPooledEther = 0;
+    rebaseLimiter.setData__harness(_fuzzData);
+
+    if (!rebaseLimiter.isLimitReached()) {
+      vm.expectRevert();
+      rebaseLimiter.getSharesToBurnLimit();
+    }
+  }
+
+  uint256 private constant MAX_ETHER_DECREASE_COEF = 1e3;
```
Comment (tamtamchik) [inline test/0.8.9/positiveTokenRebaseLimiter.t.sol:184]: Move all those constants up?

**Review by tamtamchik — APPROVED** [review]
Cool! 😎 
Thanks for uncovering some nice features!

File: test/0.8.9/positiveTokenRebaseLimiter.t.sol [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+        data.maxTotalPooledEther,
+        preTotalPooledEther + (preTotalPooledEther * rebaseLimit) / PositiveTokenRebaseLimiter.LIMITER_PRECISION_BASE
+      );
+    } else {
+      assertEq(data.positiveRebaseLimit, PositiveTokenRebaseLimiter.UNLIMITED_REBASE);
+      assertEq(data.maxTotalPooledEther, type(uint256).max);
+    }
+  }
+
+  function testFuzz_isLimitReached(TokenRebaseLimiterData calldata _fuzzData) external {
```
[the remark this answers is not in the document]
Comment (TheDZhon) [reply test/0.8.9/positiveTokenRebaseLimiter.t.sol:68]: Agree, ty ty!

File: test/0.8.9/positiveTokenRebaseLimiter.t.sol (hunk 1 above)
Comment (TheDZhon) [reply test/0.8.9/positiveTokenRebaseLimiter.t.sol:184]: Sure thing :+1:

**Review by mymphe — APPROVED** [review]

**Pull request merged**